### ISSUE TITLE
Comparison with yogiben's Admin package

### ISSUE THREAD

**Issue by dandv** [Strngs/comet#12]
You're familiar with @yogiben's package... https://github.com/yogiben/meteor-admin/issues/61

How do these two compare?

**Comment by WillHall** [comment]
I was originally writing code for admin - but then I decided to make my own fork of the admin package. Admin is great, but for my own purposes I need much more control, and that is the intent of Comet. To be feature rich, and offer lots of extensibility. 

The version available on atmosphere is admin + reactive datatables integration - the configuration is similar but requires a bit more to get all the functionality working. It is very similar to my PR https://github.com/yogiben/meteor-admin/pull/58 for admin, but also includes all of the UI changes seen in the screenshots for Comet.

Here is an example configuration file showing the syntax for CometConfig. https://github.com/WillHall/meteor-starter/blob/master/both/config/adminConfig.coffee

You will notice one of the biggest changes is the requirement to define your edit and delete columns now. As well as the render callback pointer.

There are currently many core helpers written to assist in displaying data, however, you can also easily define your own methods for display data. https://github.com/WillHall/meteor-admin/blob/master/lib/both/AdminDashboard.coffee (see formatters)

I believe @yogiben still intends to pull the PR in, but it was recently blocked by the merging of another PR that changes the pub/sub labels so it may be a bit before those issues can be resolved.

That pull request recently merged into Admin is not included in Comet, because I deemed it redundant due to datatables already handling that functionality.

When Comet reaches 2.0 I will publish a full doc set explaining all of the ins and outs - until then, I wouldn't advise using it in production as it will drastically change between then and now. I currently have 9 features left before 2.0.

There you have it, you are all caught up. Thanks for your interest. :)
